### DISCUSSION TITLE
Edit break times in lazer beatmap editor.

### DISCUSSION THREAD

**Discussion by SnowfallWolf** [ppy/osu#25825] [Ideas] [answered]
Hey so I was working on my map I imported over from stable, and noticed I can't change any break times. A way to do this would be great. Also this has a weird little bug where you can have a break time with notes going.

https://github.com/ppy/osu/assets/71610202/1d9fe344-baee-49fd-bc82-f5558feda4b9

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/14330
